src/hmrc-design-patterns/__tests__/ask-the-user-for-their-consent.integration.js

```diff
@@ -36,7 +36,7 @@ describe('Design pattern page rendering', () => {
   })
 
   describe('Copy snippet button', () => {
-    const buttonSelector = '#example-yes-no-question-html .app-link--copy'
+    const buttonSelector = 'button.app-copy-button'
```

Comment:
It would be better to keep an ID in this selector. It looks like the ID might have been changed to `example-yes-no-question` so you could use `#example-yes-no-question .app-copy-button`